Makefile

```diff
@@ -176,6 +176,9 @@ test-symbolicator:
 	@echo ""
 
 test-acceptance: node-version-check
+	echo $(PERCY_PARALLEL_NONCE)
+	echo $(PERCY_PARALLEL_NONCE2)
+	echo $(PERCY_PARALLEL_NONCE3)
```

Comment:
If you have 2 groups will you have 3 nonces?

Reply:
@markstory I was just debugging with this PR, but will bring this conversation back to the original PR.